### DISCUSSION TITLE
بک آپ گیری از مرزبان

### DISCUSSION THREAD

**Discussion by SaintShit** [Gozargah/Marzban#62] [Show and tell]
فایل دیتابیس مرزبان در مسیر
`/var/lib/marzban/`
ذخیره میشه. شما میتونید فاید های این پوشه رو کپی کنید و به همین مسیر توی سرور جدید انتقال بدید.
همینطور شاید نیاز باشه از فایل های کانفیگتون `xray_config.json` و `env` هم بک‌آپ تهیه کنید تا تغییراتتون رو به سرور جدید منتقل کنید.

**Comment by iamtheted** [comment]
درود بر سینت 💙
با سکریپت زیر میشه از همه فایل های لازم بکاپ اتوماتیک گرفت.
```bash
bash <(curl -Ls https://github.com/iamtheted/Marzban-Backup/raw/main/backup.sh)
``` 
github source: https://github.com/iamtheted/Marzban-Backup

**Reply by mehrshadmadani** [reply]
عزیز با این اسکریپت فایل مربوط به لینک های سابسکریشن هم کپی میشه ؟ که اگه یوقت پنل رو جابجا کردیم فورا لینکای سابسکریشن بالا بیان و قطع نشن؟

**Comment by shamloo1353** [comment]
درود 
بات توکن میخواد از کجا بیارم؟

**Reply by hooshedigital** [reply]
داداش سلام به ربات بات فادر باید پیام بفرستی در صورت کمک به ایدیم پیام بده @avangadmin

**Comment by mehrshadmadani** [comment]
از

BOTFother

‫‪shamloo1353‬‏ ***@***.***‬‏> در تاریخ شنبه ۲۹ ژوئن ۲۰۲۴
ساعت ۱۵:۴۳ نوشت:‬

**Comment by Eslender73** [comment]
فایل mysql که مربوط به بکاپ mysql هستش با چه دستوری همون لحظه بکاپ گرفته میشه؟
